lineman/app/components/group_page/trial_card/confirm_gift_plan_modal/confirm_gift_plan_modal.coffee

```diff
@@ -1,9 +1,13 @@
 angular.module('loomioApp').factory 'ConfirmGiftPlanModal', ->
   templateUrl: 'generated/components/group_page/trial_card/confirm_gift_plan_modal/confirm_gift_plan_modal.html'
   size: 'confirm-gift-plan-modal'
-  controller: ($scope, group) ->
+  controller: ($scope, group, ModalService, ChoosePlanModal, DonationModal) ->
```

Comment:
Note to self: at some point we should really move away from needing to inject the ModelService _and_ the modal we want to open.